nextflow_schema.json

```diff
@@ -11,6 +11,11 @@
             "description": "",
             "default": "",
             "properties": {
+                "run_annotation_tool": {
+                    "type": "string",
+                    "default": "prodigal",
+                    "description": "Assign which annotation tool to run either prodigal or prokka"
```

Comment:
Should add an enum here, but I guess we cand o that later when we review the docs

Reply:
sounds good